frontend/src/static/js/v_authorship.js

```diff
@@ -32,6 +32,8 @@ window.vAuthorship = {
     return {
       isLoaded: false,
       files: [],
+      isLoadingMoreFiles: false,
```

Comment:
one question: is `isLoadingMoreFiles` the inverse of `hasAllLoaded`?

Reply:
yup, except `hasAllLoaded` is only toggled once to avoid triggering too many dom updates from `isLoadingMoreFiles`